test_monitoring_after_rebooting_node_where_mgr_is_running is failing due to timeout while checking ceph health on prometheus pod

AssertionError: Ceph health status metrics are not updated after the rebooting node where the mgr running
assert False
 +  where False = check_ceph_health_status_metrics_on_prometheus(mgr_pod='rook-ceph-mgr-a-fbdc6f584-7c8ft', threading_lock=<unlocked _thread.RLock object owner=0 count=0 at 0x7f46fe58d690>)


Logs: https://jenkins-csb-odf-qe-ocs4.dno.corp.redhat.com/job/qe-deploy-ocs-cluster/47748/testReport/junit/tests.functional.workloads.ocp.monitoring.test_monitoring_on_negative_scenarios/TestMonitoringBackedByOCS/test_monitoring_after_rebooting_node_where_mgr_is_running/

--- a/tests/functional/workloads/ocp/monitoring/test_monitoring_on_negative_scenarios.py
+++ b/tests/functional/workloads/ocp/monitoring/test_monitoring_on_negative_scenarios.py
@@ -42,7 +42,7 @@ from ocs_ci.framework.pytest_customization.marks import (
 log = logging.getLogger(__name__)
 
 
-@retry(AssertionError, tries=30, delay=3, backoff=1)
+@retry(AssertionError, tries=60, delay=3, backoff=1)
 def wait_to_update_mgrpod_info_prometheus_pod(threading_lock):
     """
     Validates the ceph health metrics is updated on prometheus pod

